packages/compass-collection-stats/src/components/collection-stats/collection-stats.jsx

```diff
@@ -10,14 +10,14 @@ class CollectionStats extends Component {
   static displayName = 'CollectionStatsComponent';
 
   static propTypes = {
-    documentCount: PropTypes.string.isRequired,
-    totalDocumentSize: PropTypes.string.isRequired,
-    avgDocumentSize: PropTypes.string.isRequired,
-    indexCount: PropTypes.string.isRequired,
-    totalIndexSize: PropTypes.string.isRequired,
-    avgIndexSize: PropTypes.string.isRequired,
-    isReadonly: PropTypes.bool.isRequired,
-    isTimeSeries: PropTypes.bool.isRequired
+    documentCount: PropTypes.string,
```

Comment:
Does everything work if something is missing or we need to skip rendering?

Reply:
It'll all work 👍